frontend/model/contracts/group.js

```diff
@@ -1262,8 +1337,11 @@ sbp('chelonia/defineContract', {
       process ({ meta }, { state, getters }) {
         const period = getters.periodStampGivenDate(meta.createdDate)
         const current = getters.groupSettings?.distributionDate
-
-        if (current !== period) {
+        const inWaitingPeriod = !current || new Date().toISOString() < current
+        // Maybe we're updating the distribution date while in the waiting period.
+        if (inWaitingPeriod && meta.createdDate !== current) {
```

Comment:
`new Date()` is never allowed inside of `process` functions (remember?)

Please check out PR #1643 for how to this, and also please merge `master` as that PR was recently merged.